tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{36,37,38,39,310,311,312}, coverage, docs, style
+envlist = py{38,39,310,311,312}, coverage, docs, style
```

Comment:
3.6 and 3.7 are still tested via github actions (`tests.yml` -> `test-eol:`), but not by default when one runs `tox` locally (one can still do `tox -e py36` explicitly)